### PR TITLE
WheelPicker - render tests

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
*Enter description to help the reviewer understand what's the change about...*

## Changelog
*Add a quick message for our users about this change (include Compoennt name, relevant props and general purpose of the PR)*

### CODE REVIEW

File: src/incubator/WheelPicker/__tests__/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    it('should present 7 rows', () => {
+      const {getByTestId} = render(<WheelPicker {...props}/>);
+      const flatList = getByTestId('wheel.list');
+
+      expect(flatList.props.height).toBe(NUM_OF_ROWS * ITEM_HEIGHT);
+    });
+  });
+
+  //   describe('Item', () => {
```
Comment (lidord-wix) [inline src/incubator/WheelPicker/__tests__/index.spec.js:44]: Add a TODO for what you've been trying todo :)

**Comment by Inbal-Tish** [comment]
@lidord-wix I'm waiting for this PR to add another render test for my bug fix. Can you please make this a priority?

File: src/incubator/WheelPicker/__tests__/index.spec.js (hunk 1 above)
Comment (Inbal-Tish) [reply src/incubator/WheelPicker/__tests__/index.spec.js:52]: haha

File: src/uilib-test-renderer/helper.ts [hunk 2]
```diff
@@ -12,4 +13,8 @@ const findStyle = <T>(key: string, component: Component): T => {
   return StyleSheet.flatten(component.props.style)[key];
 };
 
-export {findStyle};
+const FireOnMomentumScrollEnd = (component: any, {x, y}: {x?: number, y?: number}) => {
```
Comment (Inbal-Tish) [inline src/uilib-test-renderer/helper.ts:16]: You need to use lower case for functions.
Also, 'helper' is a very general name. I would keep this file for scrollView helper functions.
The 'nativeEvent' configuration I would move to a separate function, as you might add more helper functions that will need it, like 'onScroll'.

File: src/incubator/WheelPicker/__tests__/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      const {getByTestId} = render(<TestCase/>);
+      const flatList = getByTestId('wheel.list');
+
+      expect(flatList.props.height).toBe(NUM_OF_ROWS * ITEM_HEIGHT);
+    });
+  });
+
+  //TODO: make this test work
+  //   describe('Item', () => {
+  //     it('should color selected index with activeColor', () => {
```
Comment (Inbal-Tish) [inline src/incubator/WheelPicker/__tests__/index.spec.js:53]: This description is not very clear on what you want to test (where is the verb in this sentence?)

File: src/incubator/WheelPicker/__tests__/index.spec.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+const TestCase = props => {
+  return (
+    <WheelPicker
+      testID={'wheel'}
+      items={_.times(60, i => i).map(item => ({label: `${item}`, value: item}))}
+      onChange={onChangeMock}
+      numberOfVisibleRows={NUM_OF_ROWS}
+      itemHeight={ITEM_HEIGHT}
+      initialIndex={0}
+      activeTextColor={'red'}
```
Comment (Inbal-Tish) [inline src/incubator/WheelPicker/__tests__/index.spec.js:20]: Please use uilib colors (Color.red30, Colors.blue30)

File: src/incubator/WheelPicker/__tests__/index.spec.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+describe('WheelPicker', () => {
+  describe('FlatList', () => {
+    it('should call onChange callback after scrolling', () => {
+      const {getByTestId} = render(<TestCase/>);
+      const flatList = getByTestId('wheel.list');
+
+      FireOnMomentumScrollEnd(flatList, {y: 200});
+
+      expect(onChangeMock).toHaveBeenCalled();
```
Comment (Inbal-Tish) [inline src/incubator/WheelPicker/__tests__/index.spec.js:35]: this is a redundant test as the next test coves it already.

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**